src/k8s/types/v1alpha1/emailpolicy.ts

```diff
@@ -6,26 +6,22 @@ export type SMTPSink = {
         name: string;
         namespace?: string;
     };
-    credentials: {
+    credentials?: {
         name: string;
         namespace?: string;
     };
 }
 
-export type CatchSink = {
-    retentionDays?: number;
-}
-
 export type EmailPolicySpec = {
     default?: boolean;
     podSelector?: LabelSelector;
     ratelimiting?: {
         maximum: number;
         period?: "hour" | "minute";
     };
-    sink: {smtp: SMTPSink} | {catch: CatchSink};
```

Comment:
Can we drop the nested `smtp` key here? I guess `smtp` will stay the only supported sink.

Reply:
I was hesitant to change this tbh, in case we change our mind about this at some point in the future and want to add more supported sinks again. :shrug: